bin/ask-smapi.js

```diff
@@ -13,7 +13,7 @@ if (!process.argv.slice(2).length) {
     commander.parseAsync(process.argv)
         .then(result => Messenger.getInstance().info(result))
         .catch(err => {
-            Messenger.getInstance().error(jsonView.toString(err.response));
+            Messenger.getInstance().error(jsonView.toString(err));
```

Comment:
should we do 'err.response || err' to be backward compatible? if we dont need backward compatibility in error messages, this fix is good enough;

Reply:
I think it's not a backward incompatible change, compared to before, we just add more information with this change (like headers statusCode etc.) which makes some sense when the error is not from smapi happens. It's more transparent to some degree. So I think we can just show the entire error.